cli/command_snapshot_create.go

```diff
@@ -105,6 +112,18 @@ func runSnapshotCommand(ctx context.Context, rep repo.RepositoryWriter) error {
 	return errors.Errorf("encountered %v errors:\n%v", len(finalErrors), strings.Join(finalErrors, "\n"))
 }
 
+func getTags() (map[string]string, error) {
+	tags := map[string]string{}
+	for _, tagkv := range *snapshotCreateTags {
+		tagFields := strings.Split(tagkv, ":")
```

Comment:
nit: you can also do strings.SplitN(tagkv, ":", 2) this will keep the 2nd part unsplit, which allows it to have `:`